File: C/10.21/mario.c
//Paola Frunzio
//9.30.2020
//make the hashtags appear in a pyramid based on a height input

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height, row, x;
    row = 1;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 | height > 8);

    do
    {
        //left spaces
        x = height - row - 1;
        if (x > -1)
        {
            do
            {
                printf(" ");
                x --;
            }
            while (x > -1);
        }

        //left #
        x = 1;
        do
        {
            printf("#");
            x ++;
        }
        while (x <= row);

        //middle spaces
        printf("  ");

        //right #
        x = 1;
        do
        {
            printf("#");
            x ++;
        }
        while (x <= row);

        printf("\n");
        row ++;
    }
    while (row <= height);
}

File: Python/1.11/mario.py
# Paola Frunzio
# 1.11.2021
# mario project

from cs50 import get_int

height = 0
while(height < 1 or height > 8):
    height = get_int("Height: ")

row = 1
while (row <= height):
    # left spaces
    x = height - row - 1
    while(x > -1):
        print(" ", end="")
        x -= 1

    # left #
    x = 1
    while(x <= row):
        print("#", end="")
        x += 1

    # middle spaces
    print("  ", end="")

    # right #
    x = 1
    while(x <= row):
        print("#", end="")
        x += 1
    
    print()
    row += 1